src/series/point.js

```diff
@@ -47,11 +46,20 @@ export default function() {
         decorate = x;
         return point;
     };
-    point.radius = function(x) {
+
+    point.size = function(x) {
+        if (!arguments.length) {
+            return size;
+        }
+        size = x;
```

Comment:
It looks like you are using `d3.functor` in the wrong place. It should be applied within the property implementation